### ISSUE TITLE
Crafted Items are not getting named

### ISSUE THREAD

**Issue by BarryTheShen** [mehboss/CustomRecipes#5]
Im making some custom items, but the name part in the config file seems to be not working

**Comment by mehboss** [comment]
Hello BarryTheShen! Sorry for the issues you are encountering. Could you please reply back with this applicable information:

- Startup log via pastebin.com
- Copy of the recipe file in question
- How to replicate this issue (i.e any hints or information regarding what is happening and what is intended to be happening)
- Any errors in console associated with custom recipes
- Offline/Online mode? Server version (i.e paper or spigot)

This will allow me to dig deeper into your issue and provide a solution for you.

Thank you!

**Comment by BarryTheShen** [comment]
Hello, the bug occurred all the time. I tried to reset the config files by removing them all and regenerating them which didn't work. There is 1 more thing which is the custom identifier crafting detector is not working. So when I. Satisfy the requirements of crafting with a custom item it does not think there is a crafting recipe.

**Comment by BarryTheShen** [comment]
My server is running on paper 1.20 with essentials, geyser, executable items, inventory roll back

**Comment by BarryTheShen** [comment]
The server is in offline mode

**Comment by BarryTheShen** [comment]
It seems like the custom recipe output only has the recipe working. Not anything else such as the names, enchantments, and potion effects

**Comment by mehboss** [comment]
Hello, sorry for the late reply. It seems offline-mode is causing the issue with the plugin. I will look into it further and come back here with information; however, I try not to support offline-mode because it is often a result of piracy.

**Comment by mehboss** [comment]
Hello again, I did some testing, and it seems there has been an issue with the latest release of 1.20. I will look into this and come out with a fix. I will keep you updated. Thanks for reporting!

**Comment by mehboss** [comment]
Hello, BarryTheShen!

I apologize for the wait on this fix. Please refer [here](https://www.spigotmc.org/resources/%E2%96%BA%E2%96%BA-custom-crafting-recipes-1-8-x-1-20-x-per-recipe-perms-create-your-own-recipes-%E2%97%84%E2%97%84.36925/update?update=504706) for a resolution. If this problem persists or you encounter a different issue, please open another ticket. Thank you!

Ticket closed with status of resolved.
